src/examples/graph.js

```diff
@@ -510,9 +535,10 @@ class Graph extends React.Component<IGraphProps, IGraphState> {
       });
     } else if (copiedNodes !== null) {
       // Paste multiple nodes is used if allowMultiselect is true
-      let cornerX, cornerY;
+      let cornerX;
+      let cornerY;
 
-      copiedNodes.forEach(copiedNode => {
+      Array.from(copiedNodes.values()).forEach(copiedNode => {
```

Comment:
Can you just use `copiedNodes.forEach(...`? 
The first argument passed to forEach on Map should be value and it doesn't seem like you need a new array here since the array is not assigned to anything.

Reply:
Thanks.